ui/app/templates/vault/cluster/access/oidc/assignments/assignment/details.hbs

```diff
@@ -0,0 +1 @@
+details
```

Comment:
This is a placeholder. I've already started on the details view for the next pr.